src/InteractiveWindow/Editor/IInteractiveWindowOperations2.cs

```diff
@@ -10,6 +10,12 @@ public interface IInteractiveWindowOperations2 : IInteractiveWindowOperations
         void Copy();
 
         /// <summary>
+        /// Copies only user inputs to clipboard. 
```

Comment:
From the name and description, it's not clear whether standard input will be copied.